client/my-sites/email/email-management/titan-manage-mailboxes/index.jsx

```diff
@@ -103,7 +104,7 @@ class TitanManageMailboxes extends Component {
 				{ selectedSite && hasSubscription && <QuerySitePurchases siteId={ selectedSite.ID } /> }
 
 				<Main wideLayout={ true }>
-					<DocumentHead title={ translate( 'Manage All Mailboxes' ) } />
+					<DocumentHead title={ titleCase( translate( 'Manage all mailboxes' ) ) } />
```

Comment:
This string has already been translated so there is limited benefit in using `titleCase()` here but at least we'll be consistent with other pages.